### DISCUSSION TITLE
It's my first time using this, so I don't know how to fix this problem...

### DISCUSSION THREAD

**Discussion by ySergi** [SubtitleEdit/subtitleedit#7092] [Q&A]
![image](https://github.com/SubtitleEdit/subtitleedit/assets/138529583/d96161a6-2a00-434f-9135-b6c12933b886)

As you can see, when I start the OCR it changes all the text by random characters. How do I fix this? Sorry if it's an stupid question, as I said, it's my first time using this. I want to convert some SUB files to SRT from a AVI video, as I intend to re-encode it to MKV and add the subtitles as SRT.

I selected the spanish dictionary so I don't understand why this isn't working.

**Comment by niksedk** [comment]
You could try using "Tesseract 5" - requires that you download the "Spanish Tesseract  model" (in program download).

**Reply by ySergi** [reply]
Thanks for answering, could you show me how? How do I download the spanish tesseract model?

**Comment by Purfview** [comment]
Max error 1.0% is too high, lower it till there is no such errors. For DVD subs I usually keep it between 0.0% and 0.2%.
